test/verify/check-connection

```diff
@@ -97,16 +97,8 @@ class TestConnection(MachineCase):
 
         # Lets crash a process and see if we get a proper backtrace in the logs
         # This helps with debugging failures in the tests elsewhere
-        if m.image in ["debian-testing", "ubuntu-1604"]:
-            binary = "cron"
-            m.execute("systemctl start cron")
-        elif m.image in ["fedora-atomic"]:
-            binary = "agetty"
-        else:
-            binary = "crond"
-            m.execute("systemctl start crond")
-        m.execute("kill -SEGV $(pgrep {0})".format(binary))
-        wait(lambda: m.execute("journalctl -b | grep 'Process.*{0}.*of user.*dumped core.'".format(binary)))
+        m.execute("bash -c 'kill -SEGV $$' || true")
+        wait(lambda: m.execute("journalctl -b | grep 'Process.*bash.*of user.*dumped core.'"))
```

Comment:
We did want to test that core dumps of a systemd owned process worked. Is there a way to do that in a general way? The problem was that various operating systems had that functionality disabled and/or broken.